DESCRIPTION

```diff
@@ -3,13 +3,13 @@ Type: Package
 Title: Deployment Interface for R Markdown Documents and Shiny Applications
 Version: 0.8.28
 Authors@R: c(
-    person("Aron", "Atkins", role = c("aut", "cre"), email = "aron@rstudio.com"),
-    person("Jonathan", "McPherson", role = "aut", email = "jonathan@rstudio.com"),
+    person("Aron", "Atkins", role = c("aut", "cre"), email = "aron@posit.co"),
```

Comment:
The CRAN folks might see this as a change in maintainer because our email addresses are changing. This is more for me than for anyone else, since I'll need to convince them that I am me.